### PR TITLE
OAuth: add docs for disableAutoLogin param

### PR DESCRIPTION
**Pull request by alexanderzobnin**
This PR contains docs for #28357 - `disableAutoLogin` param.

### CODE REVIEW

File: docs/sources/auth/overview.md [hunk 1]
```diff
@@ -125,6 +125,10 @@ Defaults to `false`.
 oauth_auto_login = true
 ```
 
+If you want to skip automatic OAuth login and sign in with username and password, add `disableAutoLogin` parameter to the login URL. Signing in with OAuth will be disabled in this case.
```
Comment (oddlittlebird) [inline docs/sources/auth/overview.md:128]:
```suggestion
If you want to skip automatic OAuth login and sign in with a username and password, then add the `disableAutoLogin` parameter to the login URL. This disables signing in with OAuth.
```

File: docs/sources/auth/overview.md [hunk 2]
```diff
@@ -125,6 +125,10 @@ Defaults to `false`.
...
+
+Example: `grafana.example.com/login?disableAutoLogin` or `grafana.example.com/login?disableAutoLogin=true`.
```
Comment (oddlittlebird) [inline docs/sources/auth/overview.md:130]:
```suggestion
Example: `grafana.example.com/login?disableAutoLogin` or `grafana.example.com/login?disableAutoLogin=true`
```

Comment (oddlittlebird) [reply docs/sources/auth/overview.md:130]: Great examples! We need more examples throughout the doc. I think they really help clarify items like this.

**Review by oddlittlebird — APPROVED** [review]
A couple minor nits, then g2g.

File: docs/sources/auth/overview.md [hunk 3]
```diff
@@ -125,6 +125,10 @@ Defaults to `false`.
 oauth_auto_login = true
 ```
 
+If you want to skip automatic OAuth login and sign in with a username and password, then add the `disableAutoLogin` parameter to the login URL. This disables signing in with OAuth.
```
Comment (alexanderzobnin) [inline docs/sources/auth/overview.md:128]: Forget about version availability note. This feature will be available in 7.4.

Comment (oddlittlebird) [reply docs/sources/auth/overview.md:128]: AH, yes. Definitely want to add that.

**Comment by mckn** [comment]
@alexanderzobnin do you know why this didn't get merged? Have an issue requesting this information to be added. https://github.com/grafana/grafana/issues/38723

**Comment by Eve832** [comment]
Is there a way to disable login for Cloud also @mckn ?

**Pull request closed without merging**